### PR TITLE
#185 transfer GUI only supports best supported psm_readers

### PR DESCRIPTION
**Pull request by jalew188**
FIX GUI issues in #185

### CODE REVIEW

File: peptdeep/webui/transfer_ui.py [hunk 1]
```diff
@@ -140,6 +141,15 @@ def _clear_user_mods():
 
 
 def show():
+    psm_type_to_ext_dict = {
+        "alphapept": ".ms_data.hdf",
+        "pfind": ".spectra",
+        "maxquant": "msms.txt",
+        "diann": "tsv",
+        "speclib_tsv": "tsv",
```
Comment (liuwenrui917) [inline peptdeep/webui/transfer_ui.py:149]: "msfragger_pepxml": "pepxml"

File: peptdeep/webui/__init__.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Starting the gui will raise by OpenMP in LLVM package
```
Comment (mschwoer) [inline peptdeep/webui/__init__.py:1]: +1 for commenting on the reasons here

File: peptdeep/webui/transfer_ui.py [hunk 3]
```diff
@@ -140,6 +141,15 @@ def _clear_user_mods():
...
+    }
+    used_psm_types = list(psm_type_to_ext_dict.keys())
```
Comment (mschwoer) [inline peptdeep/webui/transfer_ui.py:151]: (nit) `.keys` not required here

**Review by mschwoer — APPROVED** [review]
LGTM

File: peptdeep/webui/transfer_ui.py (hunk 1 above)
Comment (jalew188) [reply peptdeep/webui/transfer_ui.py:149]: pepxml (or msfragger_pepxml) is not well supported as I don't know how to extract FDR values.

File: peptdeep/webui/transfer_ui.py (hunk 3 above)
Comment (jalew188) [reply peptdeep/webui/transfer_ui.py:151]: I thought list(dict) will give us a list of tuples.

File: peptdeep/webui/transfer_ui.py (hunk 1 above)
Comment (jalew188) [reply peptdeep/webui/transfer_ui.py:149]: Added

**Pull request merged**